### DISCUSSION TITLE
Icons that aren't in /icons

### DISCUSSION THREAD

**Discussion by hawwokitty** [React95/React95#456] [Q&A]
Hi! I'm trying to use this in a react project but I can't seem to figure out how to get the "help" and "close" icons to be the same as here:
https://react95.github.io/React95/?path=/story/titlebar--complete

The example code just uses "`<img src={Help} alt="help" />`" but I have no idea where they're getting the {Help} from? Does anyone know?

Thanks in advance!

**Comment by ggdaltoso** [comment]
The icons inside the `@react95/icons` package come from Windows 95 itself; I couldn't find those icons there. This is why I added them as assets inside `@react95/core` package.

But, I revisited the Modal and TitleBar components and they might address your needs: #462. Can you take a look, please?

**Reply by hawwokitty** [reply]
i didnt see this until now, but yeah that has fixed the issue!
